tests/fixtures/MergeA.jsx

```diff
@@ -5,6 +5,7 @@ import { GetText } from 'gettext-lib';
 const MergeA = () =>
   <div>
     <GetText message="I'm both here and elsewhere" />
+    <GetText message="I'm both here and elsewhere" />
```

Comment:
What is the diff here?

Reply:
Oh, I was intending to add some test for line/column number. I moved the test to use another file, to avoid this kind of mixed useage.